### ISSUE TITLE
Change name 'Basic' to 'Visual Basic' in Tools Options

### ISSUE THREAD

**Issue by kendrahavens** [dotnet/roslyn#34990]
I've noticed users mistaking the 'Basic' Text Editor Tools Options to mean general .NET settings as opposed to 'Visual Basic' specific settings multiple times. Can we rename this to 'Visual Basic' to reduce confusion for new users? It's also doesn't follow brand guidelines for the language.

![image](https://user-images.githubusercontent.com/8293542/56166199-d0d09680-5f89-11e9-8435-57c36e586b3a.png)
![image](https://user-images.githubusercontent.com/8293542/56166253-fe1d4480-5f89-11e9-9417-eda5c74fe7b8.png)

**Comment by KexyBiscuit** [comment]
Or we could rename it to [BASIC](https://en.wikipedia.org/wiki/BASIC), all capitalized.

**Comment by paul1956** [comment]
Ignoring the trademark, it is confusing what setting are for VB or Visual Basic or BASIC and just basic settings that apply to everything and how to get everything that applied to VB.Net or how to configure some of the items that are hidden from the Visual Studio UI, for example when you search for formatting nothing shows up under Basic.
![image](https://user-images.githubusercontent.com/12789369/56180862-fa0c1980-5fbf-11e9-847c-62a1c7ea61c4.png)
![image](https://user-images.githubusercontent.com/12789369/56180884-0ee8ad00-5fc0-11e9-9fd7-8eb2e844339a.png)
![image](https://user-images.githubusercontent.com/12789369/56180980-73a40780-5fc0-11e9-90b8-bdf6520a8c27.png)

There are even more options for C# that don't fit but you only need one obvious search.
![image](https://user-images.githubusercontent.com/12789369/56180925-32135c80-5fc0-11e9-84ec-00de0f0828e6.png)

**Comment by jinujoseph** [comment]
cc @kendrahavens

**Comment by mikadumont** [comment]
**Feature Goals**: 
Keyboard setup to be less confusing and consolidate the settings wizard experience with the reset keyboard experience. 

**Proposal**: 
1. 'Ok' Saves the keymap profile so instead of it saying 'Ok' have it say 'Save':
![image](https://user-images.githubusercontent.com/46729679/60142359-47e69000-976e-11e9-874c-c2dab52c7e2d.png)

2. 'Reset' resets to default but if a user sets their keymap profile to a non default option and selects 'Reset' they can think its resetting to the keymap profile they chose - perhaps we can surface a message when you hover over 'Reset' to say: 'Reset to default':
![image](https://user-images.githubusercontent.com/46729679/60142479-d9ee9880-976e-11e9-84d1-d5c6054c88ae.png)

3. 'Reset' doesn't seem to fully reset keymap. For instance, a user that uninstalls a third party extension that changed their keymap profile still needs to go into the Import and Export Settings Wizard to actually reset their keymap profile back to default because 'Reset' doesn't seem to do the trick:
![image](https://user-images.githubusercontent.com/46729679/60142531-1ae6ad00-976f-11e9-8457-73005c738786.png)

4. Update keymap profile options since some are outdated: 
![image](https://user-images.githubusercontent.com/46729679/60142703-dad3fa00-976f-11e9-88c4-90c6e4f71ee2.png)

5. Make this a standard search box and have it say 'search' commands containing instead of 'show':
![image](https://user-images.githubusercontent.com/46729679/60142833-6f3e5c80-9770-11e9-84db-e915f79053df.png)

6. Commands can be grouped as: Editor Actions, Main menu, Tool Windows, Version Control. Then each option can expand like a tree view and list the commands that fall under that section. Also command names can be written more clearly: For example: 'Edit.ToggleBlockComment' can simply be 'Toggle Block Comment'.

7. Add tags for search indexing for the following words: shortcut, keybindings, keybind and keymap.

**Comment by jinujoseph** [comment]
**Design Meeting Notes**
1. Let's do it,  Rename "Basic"  in the tools -> options to "Visual Basic" 
2. In the keyboard mapping, today we have Visual Basic 6  -> this need to be renamed to "Visual Basic" 
**Note**: these pre-dated the profiles that were added in VS2005, and were there in VS 2002 and 2003 for people that were coming from VB6 or VC6.  C#’s was added in 2005 when we first designed the C# profile and keyboard shortcuts, completely ok to rename to just say Visual Basic now.
